controllers/session/validation.go

```diff
@@ -0,0 +1,66 @@
+package session
+
+import (
+	"strconv"
+
+	"emperror.dev/errors"
+
+	istiov1alpha1 "github.com/maistra/istio-workspace/api/maistra/v1alpha1"
+	"github.com/maistra/istio-workspace/pkg/model"
+)
+
+// Validator returns a string of Type and a possible error.
+type Validator func(store model.LocatorStatusStore) (string, error)
+
+func chainValidator(ctx model.SessionContext, ref model.Ref, session *istiov1alpha1.Session, validators ...Validator) model.ModificatorController {
+	return func(store model.LocatorStatusStore) bool {
+		hasError := false
```

Comment:
if we would instead define `succeeded := true` we could use that for L26 and return in L45. maybe a bit more readable and coherent?

Reply:
Good point! https://github.com/maistra/istio-workspace/pull/866/commits/717a8a18052daef9f445694ed2b1b551dfcb23d0